pytest_html/plugin.py

```diff
@@ -278,16 +282,24 @@ def append_other(self, report):
         self._appendrow('Rerun', report)
 
     def _generate_report(self, session):
+        ansi_css = [
+            '\n/******************************',
+            ' * ANSI2HTML STYLES',
+            ' ******************************/\n']
```

Comment:
Nit: Could you move this down to line 300 so it's immediately before where it's used. I think this makes it a little easier when reading the code.